platform/src/pages/id/response_report/components/response_summary.rs

```diff
@@ -128,41 +128,62 @@ pub fn ResponseSummary(props: ResponseSummaryProps) -> Element {
+                                        colors: survey.colors.clone(),
+                                    }
                                 }
                             }
                             div {
                                 class: "flex flex-col w-[700px] justify-start items-start mb-[30px]",
-                                div {
-                                    class: "flex flex-row w-full h-[40px] justify-start items-center bg-[#edeeee]",
+                                if survey.colors.len() != 0 {
                                     div {
-                                        class: "flex flex-row w-[350px] h-full justify-start items-center border border-white px-[10px]",
-                                        {props.item.clone()}
+                                        class: "flex flex-row w-full h-[40px] justify-start items-center bg-[#edeeee]",
+                                        div {
+                                            class: "flex flex-row w-[350px] h-full justify-start items-center border border-white px-[10px]",
+                                            {props.item.clone()}
+                                        }
+                                        div {
+                                            class: "flex flex-row w-[350px] h-full justify-start items-center border border-white px-[10px]",
+                                            {props.reply.clone()}
+                                        }
                                     }
-                                    div {
-                                        class: "flex flex-row w-[350px] h-full justify-start items-center border border-white px-[10px]",
-                                        {props.reply.clone()}
+                                    for (i, label) in survey.labels.iter().enumerate() {
+                                        div {
+                                            class: "flex flex-row w-full h-[40px] justify-start items-center bg-white border border-b-[#e2e3e4] border-t-transparent border-r-transparent border-l-transparent",
+                                            div {
+                                                class: "flex flex-row w-[350px] h-full justify-start items-center px-[10px]",
+                                                {label.clone()}
+                                            }
+                                            div {
+                                                class: "flex flex-row w-[350px] h-full justify-between items-center px-[10px]",
+                                                div {
+                                                    {format!("{}%", survey.value_percents[i])}
+                                                }
+                                                div {
+                                                    {format!("{}", survey.value_counts[i])}
```

Comment:
```suggestion
                                                    "{survey.value_counts[i]}"
```
이렇게도 되지 않던가요? 여기에서는 기본적으로 format! 이 들어가는 상태가 됩니다.

Reply:
아 넵 되네요. 수정했습니다!